scaleway/table_scaleway_invoices.go

```diff
@@ -0,0 +1,259 @@
+package scaleway
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"time"
+
+	billing "github.com/scaleway/scaleway-sdk-go/api/billing/v2beta1"
+
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableScalewayInvoices(ctx context.Context) *plugin.Table {
+	plugin.Logger(ctx).Debug("Initializing Scaleway invoices table")
+	return &plugin.Table{
+		Name:        "scaleway_invoices",
+		Description: "invoices in your Scaleway account.",
+		List: &plugin.ListConfig{
+			Hydrate: listScalewayInvoices,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "organization_id", Require: plugin.Optional},
```

Comment:
Could we include the `type`, `start_date`, and `stop_date` columns as an optional qualifier? The API supports this parameter as input, and it would provide users with improved filtering capabilities.